sdk-autogen/pom.xml

```diff
@@ -63,43 +63,40 @@
         <executions>
           <execution>
             <id>getCommonProto</id>
-            <!-- the wget goal actually binds itself to this phase by default -->
```

Comment:
why are the comments removed ?

Reply:
Because, at least according to the docs page where I found the cache directory / config settings for the wget plugin, this is an inaccurate statement. See http://maven-download-plugin.github.io/maven-download-plugin/docsite/1.6.0/wget-mojo.html. The docs say "Binds by default to the lifecycle phase: process-resources."